src/Components/WishlistCard/WishlistCard.jsx

```diff
@@ -1,29 +1,51 @@
 import React from "react";
 import "./WishlistCard.css";
 import { Link } from "react-router-dom";
-export function WishlistCard() {
+import { useWishlist } from "../../context/wishlist-context";
+import { useCart } from "../../context/cart-context";
+export function WishlistCard(items) {
+  const { cart, addToCart } = useCart();
+  const { wishlist, addToWishlist, removeFromWishlist } = useWishlist();
+  const { image, title, description, price, rating } = items;
+  console.log(items);
+
+  function moveToCart(product) {
+    addToCart(product);
+    removeFromWishlist(product);
+  }
   return (
     <div className="product-content-container">
       <div className="product-list">
         <div className="Wishlist-container">
           <div className="card2-container">
-            <Link to="#" className=" wishlist-card card-closebtn">
-              &times;
-            </Link>
-            <img
-              src="https://img.tatacliq.com/images/i7/658Wx734H/MP000000010223017_658Wx734H_202108092243441.jpeg"
-              alt="image-bag"
-            />
-            <h1>Michael Kors</h1>
-            <h2>Karlie Medium Leather Satchel</h2>
-            <h5>
-              <ins>₹22,500</ins>
-              <del>₹45,000</del>50% Off
-            </h5>
+            <img src={image} alt="image-bag" />
+            <h1>{title}</h1>
+            <h2>{description}</h2>
+            <h5>{price}</h5>
             <div class="divider-sidebar"></div>
-            <Link to="/cart" className="btn-bag-card">
-              <span>MOVE TO BAG</span>
-            </Link>
+            {wishlist.find((item) => item._id === items._id) ? (
+              <i
```

Comment:
Same here. You can create a separate component

Reply:
I am not sure about it. Will read about it. Thanks